index.html

```diff
@@ -8,61 +8,98 @@
     <title>Company Site</title>
     <META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW" />
   </head>
+
   <body>
-    <nav>
-      <a class="logo" href="#">
+    <header class="header">
+      <a class="header--logo" href="#">
         <img src="images/keys.png" alt="keyboard" />
       </a>
-      <ul class="nav-links-cont">
-        <a class="nav-links" href="#">
-          <li>Link</li>
+
+      <nav class="header--nav">
+        <a class="header--link" href="#">
+          Link
         </a>
-        <a class="nav-links" href="#">
-          <li>Link</li>
+
+        <a class="header--link" href="#">
+          Link
         </a>
-        <a class="nav-links" href="#">
-          <li>Link</li>
+
+        <a class="header--link" href="#">
+          Link
         </a>
-        <a class="nav-links" href="#">
-          <li>Link</li>
+
+        <a class="header--link" href="#">
+          Link
         </a>
-        <a class="nav-call" href="#">
-          <button>Button</button>
+
+        <a class="button--small header--button">
+          Button
         </a>
-      </ul>
-    </nav>
-    <div class="top-cont">
+      </nav>
+    </header>
+
+    <div class="main">
       <div class="gradient-overlay"></div>
-      <div class="image-and-call-cont">
-        <div class="call-cont">
-          <div class="call-cont-inner">
-            <h1>Web development at its finest</h1>
-            <a href="#">
-              <button>Button</button>
-            </a>
+
+      <div class="hero">
+        <div class="hero--cta">
+          <div class="hero--content">
+            <h1 class="hero--heading">
+              Web development at its finest
+            </h1>
+
+            <button class="button">
+              Button
+            </button>
           </div>
         </div>
-        <div class="img-cont"></div>
+
+        <div class="hero--image"></div>
       </div>
-      <div class="icon-row">
-        <div class="icon-cont">
-          <div class="icon-single">
-            <img src="images/keys.png" alt="keyboard" />
-            <h2>Heading</h2>
-            <p>This is a small paragraph, nothing too fancy</p>
+
+      <section class="section">
+        <div class="cards">
+          <div class="card">
+            <img class="card--image" src="images/keys.png" alt="keyboard" />
```

Comment:
Called this card--image to keep things uniform, we already used hero--image earlier so I figured card--image might just cause more confusion